io.gdevelop.ide.yml

```diff
@@ -0,0 +1,54 @@
+# This is the Flatpak manifest for GDevelop.
+app-id: io.gdevelop.ide
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
```

Comment:
does it support wayland?

Reply:
Good question, we're still on Electron 8 so I think that's a no, until the time we update to the latest Electron version.

Reply:
Then

```suggestion
  - --socket=x11
```